cosmo_tester/resources/blueprints/active-directory/active-directory-blueprint.yaml

```diff
@@ -0,0 +1,103 @@
+tosca_definitions_version: cloudify_dsl_1_2
+
+description: >
+  This blueprint defines a EC2 VM created using Cloudify's AWS plugin
+  and a simple web server started using Cloudify's script plugin.
+  In addition, an elastic IP and a security group are created and associated with the created VM.
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3/types.yaml
+  - http://www.getcloudify.org/spec/aws-plugin/1.3/plugin.yaml
+
+inputs:
+  active_directory_port:
+    description: The active directory server port
+    default: 389
+  image_id: 
+    description: AWS EC2 image id to use for the server
+    default: 'ami-e4034a8e'
+  aws_config:
+    default: {}
+  instance_type:
+    description: AWS EC2 instance type to use for the server
+    default: m3.medium
+
+node_templates:
+  elastic_ip:
+    type: cloudify.aws.nodes.ElasticIP
+    properties:
+      aws_config: { get_input: aws_config }
+
+  security_group:
+    type: cloudify.aws.nodes.SecurityGroup
+    properties:
+      aws_config: { get_input: aws_config }
+      description: Security group for active directory test VM
+      rules:
+        - ip_protocol: tcp
+          cidr_ip: 0.0.0.0/0
+          from_port: { get_property: [ active_directory_server, port ] }
+          to_port: { get_property: [ active_directory_server, port ] }
+
+  vm:
+    type: cloudify.aws.nodes.Instance
+    properties:
+      aws_config: { get_input: aws_config }
+      agent_config:
+        install_method: none
+      image_id: { get_input: image_id }
+      instance_type: { get_input: instance_type }
+      parameters: 
+        user_data: |
+          <powershell>
+            $Computer = [ADSI]"WinNT://$Env:COMPUTERNAME,Computer"
+            $bob = $Computer.Create("User", "bob")
+            $bob.SetPassword("b0b_Aa123456!")
```

Comment:
1. These are active directory users. The group they belong to is meaningless for the test since this test only tests authentication against the active directory and not authorization. Adding them to the Administrators group is just a quick way for having them added as users to the active directory later on.
2. It is supported by amazon and probably also by the plugin. I'll need to consult with Trummel